src/figma/run.ts

```diff
@@ -0,0 +1,40 @@
+import generateStylesheets from './generate-stylesheets.js';
+import getFigmaVariables from './get-figma-variables.js';
+import generateDesignTokens from './generate-design-tokens.js';
+import writeTokens from './write-tokens.js';
+
+const figmaToken = process.env.FIGMA_TOKEN ?? '';
+const figmaFileId = 'A4B1kaT5HVLqcijwK4GXzt';
+const tokensDirectory = 'tokens';
+const stylesheetsDirectory = 'stylesheets';
```

Comment:
For now I'm writing to this temporary directory so we don't clobber the current stylesheets. But in my next PR, I'll update this to `src/styles/variables`. I'll also update the `.gitignore` when that happens 👍